### PR TITLE
Week 12 project pull request from Maria Sjögren and Maja Larsson

### PR DESCRIPTION
**Pull request by molbimien**
Here's our pull request for this week's project: Build your own Movies site!

## What?

For this project, we added code for all the requirements on the blue level.  

The biggest challenge this week was to find the time to work together. We therefore decided early on to start with self-studies and to go through all the project material  on our own.  We then had a couple of live calls to set up the basic project structure and get started. Thereafter, we decided to split the tasks between ourselves. While Maja focused on getting the routing in place in the project in VS Code, Maria focused on implementing the design, creating the design for both the list and the detail page in Codesandbox.io. 

We then met up for a final live session where we put all the code together and celebrated that we were able to get the work done! 🥳

## Find the site here:

https://molbimien-maja-week-12-popular-movies.netlify.app/

### CODE REVIEW

File: code/src/App.js [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
+import List from './pages/List'
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
+    <BrowserRouter>
+      <Routes>
+      <Route exact path="/" element={<List/>}>
```
Comment (Elsa-Johanna) [inline code/src/App.js:10]: Why is this sent as an element? 

We did it like this: `<Route exact path="/" component={MainSite}>`

We are curious how it works!

File: code/src/pages/Detail.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+            .then ((res) => res.json())
+            .then ((json) => {
+                setMovie(json)
+            })
+    }, [id])
+
+    return (
+        <article className="movieDetails">
+            <Link to="/" className="backLink">
+            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30">
```
Comment (Elsa-Johanna) [inline code/src/pages/Detail.js:21]: Nice that you used this too! :D

File: code/src/pages/Detail.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+                <path d="M27 14.5C27 7.596441 21.4035594 2 14.5 2S2 7.596441 2 14.5 7.5964406 27 14.5 27 27 21.403559 27 14.5zm-19.3388348-.353553l7.4852814-7.485282c.1952622-.195262.5118446-.195262.7071068 0l2.1213203 2.121321c.1952622.195262.1952622.511844 0 .707106L12.9644661 14.5l5.0104076 5.010408c.1952622.195262.1952622.511844 0 .707106l-2.1213203 2.121321c-.1952622.195262-.5118446.195262-.7071068 0l-7.4852814-7.485282c-.19799-.19799-.197989-.509117 0-.707106z" fill="#fff" fillRule="evenodd">
+                    </path>
+            </svg>{" "}
+            Movies
+            </Link>
+            {movie && (
+                <div className="cover"
```
Comment (annaester) [inline code/src/pages/Detail.js:28]: Nice putting the backgroundimg in a div - we struggled a lot with getting the backgroundimg not to change size - responsivness. So it was nice to see that you solved it like this. 

And you put all the styling directly in the div... we tried to do it in the css..

File: code/src/pages/Icon/Back.js [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+export const BackIcon = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 30 30">
```
Comment (Elsa-Johanna) [inline code/src/pages/Icon/Back.js:4]: is this repeating itself or are they two different icons?

File: code/src/pages/Images.js [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+
+    fetch(IMAGES_URL)
+    .then ((res) => res.json())
+    .then ((json) => {
+        console.log(json.images)
+    })
+
+    return (
+        <div>
+        <p>lala</p>
```
Comment (Elsa-Johanna) [inline code/src/pages/Images.js:15]: what is this? :D

File: code/src/pages/Images.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+
+import { IMAGES_URL } from '../utils/urls'
+
+const Images = () => {
+
+    fetch(IMAGES_URL)
+    .then ((res) => res.json())
+    .then ((json) => {
+        console.log(json.images)
```
Comment (Elsa-Johanna) [inline code/src/pages/Images.js:10]: just a friendly reminder to clean the console.logs!

File: code/src/pages/Images.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import React from 'react'
```
Comment (Elsa-Johanna) [inline code/src/pages/Images.js:1]: Is this component really needed?

**Review by Elsa-Johanna — APPROVED** [review]
Hello! 

Thank you for letting us look at the code. 

There is some feedback  on your webpage that would be intersting to elaborate on Like the images component. 

BUT! Overall we are impressed with what you have achieved especially considering the time difficulties.

**Pull request closed without merging**